src/Text/Pandoc/Writers/Man.hs

```diff
@@ -270,6 +270,9 @@ inlineToMan :: PandocMonad m => WriterOptions -> Inline -> StateT WriterState m
 inlineToMan opts (Span _ ils) = inlineListToMan opts ils
 inlineToMan opts (Emph lst) =
   withFontFeature 'I' (inlineListToMan opts lst)
+-- Underline is not supported, so treat the same as Emph
+inlineToMan opts (Underline lst) =
+  withFontFeature 'I' (inlineListToMan opts lst)
```

Comment:
Here I notice you do render as Emph; would it make sense to do that in other writers where underline is not supported?

Reply:
Done.